README.org

```diff
@@ -135,6 +135,15 @@ Key bindings are, as always, ultimately up to the user.  However, in [[Configura
 
 When option ~activities-bookmark-store~ is enabled, an Emacs bookmark is stored when a new activity is made.  This allows the command ~bookmark-jump~ (~C-x r b~) to be used to resume an activity (helping to universalize the bookmark system).
 
+** Annotation and Sort
+
+When selecting an activity in the mini-buffer, annotations are sorted active first, then by most recent modification age.  If you are using an interface that supports /annotations/, additional information is shown alongside the activity name:
+
+- an `@` symbol, for active activities,
+- the number of buffers and files the activity contains,
+- a color-coded age since the activity was last updated, and
+- a final ~*~ if the activity's list of buffers and files has been modified from its default state.
```

Comment:
I think the sorting order will be obvious to the user, so probably the only thing that needs to be documented is what the non-obvious symbols mean.